### PR TITLE
refactor: extract links to constants.js

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/popup/router/pages/TermsOfService.vue [hunk 1]
```diff
@@ -72,7 +72,7 @@
         <span>{{ item.title }}</span>
       </p>
       <div
-        v-if="index == 0"
+        v-if="index === 0"
```
Comment (CedrikNikita) [inline src/popup/router/pages/TermsOfService.vue:75]: Shouldnt be a part of this commit.

**Review by CedrikNikita — COMMENTED** [review]
@Liubov-crypto can you please check if all the external links are working correctly?

**Comment by Liubov-crypto** [comment]
Tested, LGTM.

**Review by CedrikNikita — APPROVED** [review]
Code changes look reasonable.

**Pull request merged**